### ISSUE TITLE
[Bug]: Permission denied when trying to REPORT while log is open

### ISSUE THREAD

**Issue by OfficialEsco** [ThioJoe/YT-Spammer-Purge#403]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

see Steps to reproduce

### Release version

2.12.2

### Steps to reproduce

1. Scan a video with Spammers
2. Open the log file (to review the spam)
3. Forget to close the log file and type REPORT (or exclude IIRC)
4. See the error

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
==================== (See log file for channel IDs of matched authors above) ====================

 NOTE:  Check that all comments listed above are indeed spam.
 > If you see missed spam or false positives, you can submit a filter suggestion here: TJoe.io/filter-feedback

How do you want to handle the matched comments above?
~~ Not Your Channel Mode: Only Reporting is Possible ~~
 > To exclude certain authors: Type 'exclude' followed by a list of the numbers (or ranges of #'s) from the sample list
      > Example:  exclude 1, 3-5, 7, 12-15
 > To just report the comments for spam, type ' REPORT '. (Can be done even if you're not the channel owner)
 > To do nothing, simply hit Enter

Input: REPORT
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1384, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1361, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1269, in primaryInstance
  File "Scripts\logging.py", line 584, in write_log_completion_summary
  File "Scripts\logging.py", line 241, in write_rtf
PermissionError: [Errno 13] Permission denied: 'logs\\Spam_Log_2022-01-20_15-33-02.rtf'
------------------------------------------------
Error Message: [Errno 13] Permission denied: 'logs\\Spam_Log_2022-01-20_15-33-02.rtf'

Unknown Error - Code: Z-3 occurred. If this keeps happening, consider posting a bug report
on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report

 Press Enter to Exit...
```

**Comment by OfficialEsco** [comment]
That is understandable, the main issue is the `Press Enter to Exit...` instead of a retry with a warning, kinda rough when you hit a channel with 70k-100k+ comments 😁

**Comment by ThioJoe** [comment]
True haha, it was something I had thought about but hadn't bothered to add. Should be fixed with 5801d0040e4bb821c2eade8a47291b016c70fa11 for the next beta release.
